e2e/basic.e2e.ts

```diff
@@ -6,13 +6,13 @@ import {test, expect} from '@playwright/test';
   ['iframe', 'svelte'],
   ['iframe', 'vue'],
   ['iframe', 'htm'],
-  // ['iframe', 'react'],
-  // ['worker', 'vanilla'],
-  // ['worker', 'preact'],
-  // ['worker', 'svelte'],
+  ['iframe', 'react'],
+  ['worker', 'vanilla'],
+  ['worker', 'preact'],
+  ['worker', 'svelte'],
   // ['worker', 'vue'],
```

Comment:
This example is still broken, but for unrelated reasons. I will fix it in a follow-up PR.